### DISCUSSION TITLE
Informations bonne pratique

### DISCUSSION THREAD

**Discussion by victorgyt** [dbarzin/mercator#713] [Q&A]
Bonjour, 

Nous souhaitons savoir où renseigner la catégorie application (type respository) dans les Infrastructure de Stockage ? 

Je m'explique, nous disposons d'un système de stockage dédié à nos backup, et nous souhaitons lier cette infra à l'application que nous venons de créer dans Mercator (comme pour les VM et leur application). 

Où puis-je noter ces données ? Quelles sont les bonnes pratiques ? 

Merci

Cordialement 

VG

**Comment by dbarzin** [comment]
Il faut pouvoir faire un lien entre un serveur logique (dans la vue logique) et une infrastructure de stockage (dans la vue physique). Ce lien n'existe pas pour l'instant. Est-ce bien de cela dont vous avez besoin ?
